daemon/rpc_feature_toggles.go

```diff
@@ -0,0 +1,13 @@
+package daemon
+
+import (
+	"context"
+
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
+)
+
+// Returns the list of feature toggles fetched from the remote config
+// TODO (dfe): Use the actual info from the remote config
```

Comment:
Do we have a task for this or is it being worked on in a separate PR? Just to not forget it